source/common/config/grpc_stream.h

```diff
@@ -60,12 +65,13 @@ class GrpcStream : public Grpc::AsyncStreamCallbacks<ResponseProto>,
     }
     stream_ = async_client_->start(service_method_, *this, Http::AsyncClient::StreamOptions());
     if (stream_ == nullptr) {
-      ENVOY_LOG(warn, "Unable to establish new stream");
+      ENVOY_LOG(debug, "Unable to establish new stream");
```

Comment:
Isn't this one pretty important though if the config server is completely busted? (No healthy hosts, etc.)

IMO we should make this error message easier to understand:

```suggestion
      ENVOY_LOG(debug, "Unable to establish new stream to configuration server");
```

But also potentially rate limit the output? WDYT?

Reply:
https://github.com/envoyproxy/envoy/pull/14616#discussion_r555194930 aserted that this message is always accompanied by the remote close message. Tracing code (see e.g. code around [here](https://github.com/envoyproxy/envoy/blob/v1.19.0/source/common/grpc/async_client_impl.cc#L55)) this does seem to be the case at least for `AyncStreamImpl`. If you prefer to be conservative, I may just keep this as a warning for now rather than adding more logic for it

Reply:
If you are sure it always prints together I think it's fine to downgrade, but I would confirm with manual testing. The issue here is I don't think every case that this would get printed in would result in a remote close, for example no healthy host.